ldk/javascript/src/types/oliveHelps/whisper.d.ts

```diff
@@ -54,6 +55,8 @@ declare namespace WhisperService {
 
   type IconSize = 'small' | 'medium' | 'large' | 'x-large';
 
+  type Shape = 'circular' | 'linear';
```

Comment:
`Shape` is a pretty broad type, and one that we probably want to make more specific to the Progress component, just in case. So maybe it should be `ProgressShape` or something like that.